### PR TITLE
Add untyped additionalData field to LLBActionExecutionResponse proto

### PR DESCRIPTION
**Pull request by virtan**

### CODE REVIEW

File: Protos/BazelRemoteAPI/googleapis/google/rpc/error_details.proto [hunk 1]
```diff
@@ -89,36 +89,39 @@ message QuotaFailure {
...
   // domain of errors. This should be at most 63 characters and match
   // /[A-Z0-9_]+/.
   string reason = 1;
 
-  // The logical grouping to which the "reason" belongs.  Often "domain" will
-  // contain the registered service name of the tool or product that is the
-  // source of the error. Example: "pubsub.googleapis.com". If the error is
-  // common across many APIs, the first segment of the example above will be
-  // omitted.  The value will be, "googleapis.com".
+  // The logical grouping to which the "reason" belongs. The error domain
```
Comment (sergiocampama) [inline Protos/BazelRemoteAPI/googleapis/google/rpc/error_details.proto:119]: these changes appear to be unrelated to the PR, could you remove them to keep the PR focused on the changes?

Comment (virtan) [reply Protos/BazelRemoteAPI/googleapis/google/rpc/error_details.proto:119]: I didn't change it, it is what `make generate` did. Do you ask me to change it manually?

Comment (virtan) [reply Protos/BazelRemoteAPI/googleapis/google/rpc/error_details.proto:119]: Ah, I think I forgot to rebase

Comment (virtan) [reply Protos/BazelRemoteAPI/googleapis/google/rpc/error_details.proto]: Changed manually
